smartHome/src/main/java/com/cnu2016/assignment02/Main.java

```diff
@@ -0,0 +1,23 @@
+package com.cnu2016.assignment02;
+import java.io.*;
+import java.util.*;
+public class Main {
+    static final int TOTAL_APPLIANCES = 3;
+    public static void main(String args[]) throws IOException {
+        boolean state[] = new boolean[TOTAL_APPLIANCES];
+        ArrayList<Event> events = EventGenerator.readData("/projects/assignments/smartHome/input.txt");
+        Timer timer = new Timer();
+        events.forEach( e -> {
+            timer.schedule(new TimerTask() {
+                @Override
+                public void run() {
+                    state[e.ID] ^= true;
```

Comment:
This would print Home Automation state at time of scheduling, not at time of occurrence of event.